.azure-pipelines-templates/checks.yml

```diff
@@ -28,5 +28,5 @@ jobs:
     - script: |
         source env/bin/activate
         pip install cmake_format
-        cmake-format --check CMakeLists.txt $(find . -name "*.cmake" | tr "\n" " ")
+        cmake-format --check CMakeLists.txt $(find ./cmake ./samples ./src ./tests -name "*.cmake" | tr "\n" " ")
```

Comment:
We don't want to format `3rdparty` cmake files (luckily we currently have none?), or any of the generated files within our `build` directories. We get away with this on the CI because the build directories don't exist yet/in this job, but for our useage ("I want to copy paste this command but change the `--check` to `-i`"), I think we should list the expected CMake directories explicitly.